src/app/components/BookContent/BookShare.jsx

```diff
@@ -25,13 +29,13 @@ class BookShare extends React.Component {
 
     nyplStaffPick = `${shareText} ${bookTitle}`;
 
-    this.state = {
+    this.setState({
```

Comment:
linter doesn't complain about using setState(); in componentDidMount? https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-mount-set-state.md

Could possibly resolve using their recommended approach if it's a linter rule.

Reply:
True. This is using `window` for the page's url. So in the next iteration we can remove how that works and then remove `setState` from `componentDidMount`.